### ISSUE TITLE
Runing Error

### ISSUE THREAD

**Issue by EdenShadow** [microsoft/CoCosNet-v2#16]
file in archive is not in a subdirectory archive/: latest_net_D.pth

**Comment by orydatadudes** [comment]
have the same error when try to load latest_net_Corr.pth

**Comment by TianxiangMa** [comment]
Me too. Did the author upload the wrong Corr.pth file?

**Comment by EdenShadow** [comment]
> 

delete latest_net_Corr.pth

**Comment by EvgenCopyrightagent** [comment]
yes, same error, "file in archive is not in a subdirectory archive"
have anybody found solution?
